### DISCUSSION TITLE
Create .github repository

### DISCUSSION THREAD

**Discussion by jdsika** [lichtblick-suite/lichtblick#231] [General]
* move contribution guidelines & code of conduct to organization repo (.github) for it to apply to the whole organization
* create minimal organization profile which makes it clear how to get in touch with the community

**Comment by carolinabamorim** [comment]
Hi @jdsika , the community should get in touch through these Discussions tab or even send an email to the email posted in the organization profile.
Regarding the contribution guidelines and code of conduct we will open a task to move them to the organization repo.
Thank you.
